### PR TITLE
remove some more dead code and naked gl

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2ad2c4d55441ae0cce701bf669de634c7fbb0a00 available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d1e511efcb073fe6d72759b4ac928d06b2bb1847 available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7795eaace630d4e1de99092c57ef0bfc207469ab available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

File: libraries/shared/src/Transform.h [hunk 1]
```diff
@@ -87,20 +87,20 @@ class Transform {
...
 
     const Vec3& getScale() const;
     void setScale(float scale);
     void setScale(const Vec3& scale);  // [new this] = [this.translation] * [this.rotation] * [scale]
     void preScale(float scale);
     void preScale(const Vec3& scale);
-    void postScale(float scale);       // [new this] = [this] * [scale] equivalent to glScale
-    void postScale(const Vec3& scale); // [new this] = [this] * [scale] equivalent to glScale
+    void postScale(float scale);       // [new this] = [this] * [scale] equivalent to gl Scale
+    void postScale(const Vec3& scale); // [new this] = [this] * [scale] equivalent to gl Scale
```
Comment (samcake) [inline libraries/shared/src/Transform.h]: Actually no, we are talking about the function glScale()

File: libraries/shared/src/Transform.h [hunk 2]
```diff
@@ -87,20 +87,20 @@ class Transform {
...
+    void postRotate(const Quat& rotation);  // [new this] = [this] * [rotation] equivalent to gl Rotate
 
     const Vec3& getScale() const;
     void setScale(float scale);
     void setScale(const Vec3& scale);  // [new this] = [this.translation] * [this.rotation] * [scale]
     void preScale(float scale);
     void preScale(const Vec3& scale);
-    void postScale(float scale);       // [new this] = [this] * [scale] equivalent to glScale
-    void postScale(const Vec3& scale); // [new this] = [this] * [scale] equivalent to glScale
+    void postScale(float scale);       // [new this] = [this] * [scale] equivalent to gl Scale
```
Comment (samcake) [inline libraries/shared/src/Transform.h]: Actually no, we are talking about the function glScale()

File: libraries/shared/src/Transform.h [hunk 3]
```diff
@@ -87,20 +87,20 @@ class Transform {
...
     void setTranslation(const Vec3& translation); // [new this] = [translation] * [this.rotation] * [this.scale]
     void preTranslate(const Vec3& translation);   // [new this] = [translation] * [this]
-    void postTranslate(const Vec3& translation);  // [new this] = [this] * [translation] equivalent to glTranslate
+    void postTranslate(const Vec3& translation);  // [new this] = [this] * [translation] equivalent to gl Translate
 
     const Quat& getRotation() const;
     void setRotation(const Quat& rotation); // [new this] = [this.translation] * [rotation] * [this.scale]
     void preRotate(const Quat& rotation);   // [new this] = [rotation] * [this]
-    void postRotate(const Quat& rotation);  // [new this] = [this] * [rotation] equivalent to glRotate
+    void postRotate(const Quat& rotation);  // [new this] = [this] * [rotation] equivalent to gl Rotate
```
Comment (samcake) [inline libraries/shared/src/Transform.h]: Actually no, we are talking about the function glRotate()

File: libraries/shared/src/Transform.h [hunk 4]
```diff
@@ -87,20 +87,20 @@ class Transform {
     const Vec3& getTranslation() const;
     void setTranslation(const Vec3& translation); // [new this] = [translation] * [this.rotation] * [this.scale]
     void preTranslate(const Vec3& translation);   // [new this] = [translation] * [this]
-    void postTranslate(const Vec3& translation);  // [new this] = [this] * [translation] equivalent to glTranslate
+    void postTranslate(const Vec3& translation);  // [new this] = [this] * [translation] equivalent to gl Translate
```
Comment (samcake) [inline libraries/shared/src/Transform.h]: Actually no, we are talking about the function glTranslate()

**Comment by hifi-gustavo** [comment]
Build console output for commit: d5a6e87a029c2ce9c2d621b0b9e0e96b89a4a86f available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5165210e11237f101191a0864f543b56e9857c3a available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

**Comment by samcake** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5165210e11237f101191a0864f543b56e9857c3a available at: https://gist.github.com/b25097f859e9420a844c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5342.dmg

**Pull request merged**